http/base/actor/net_server.go

```diff
@@ -194,3 +195,13 @@ func GetNodeType() (uint64, error) {
 	}
 	return r.NodeType, nil
 }
+
+func SendEmergencyGovReq(msg interface{}) ontErrors.ErrCode {
+	input := &common.EmergencyMsg{
+		MsgType: common.EmergencyAdminStart,
+		Content: msg,
+	}
+	netServerPid.Tell(input)
```

Comment:
do we need check if EmergencyGovReq succeeded?

Reply:
As we talked, no need